magicctapipe/scripts/lst1_magic/config_general.yaml

```diff
@@ -0,0 +1,24 @@
+mc_tel_ids:
+    LST-1: 1
+    LST-2: 0
+    LST-3: 0
+    LST-4: 0
+    MAGIC-I: 2
+    MAGIC-II: 3
+
+directories:
+    workspace_dir : "/fefs/aswg/workspace/raniere/" #Pay attention to put the last "/"!!!
+    target_name   : "CrabTeste"
+    MC_gammas     : "/fefs/aswg/data/mc/DL0/LSTProd2/TestDataset/sim_telarray/" #Pay attention to put the last "/"!!!
+    MC_electrons  : "/fefs/aswg/data/mc/DL0/LSTProd2/TestDataset/Electrons/sim_telarray/" #Pay attention to put the last "/"!!!
+    MC_helium     : "/fefs/aswg/data/mc/DL0/LSTProd2/TestDataset/Helium/sim_telarray/" #Pay attention to put the last "/"!!!
+    MC_protons    : "/fefs/aswg/data/mc/DL0/LSTProd2/TrainingDataset/Protons/dec_2276/sim_telarray/" #Pay attention to put the last "/"!!!
+    MC_gammadiff  : "/fefs/aswg/data/mc/DL0/LSTProd2/TrainingDataset/GammaDiffuse/dec_2276/sim_telarray/" #Pay attention to put the last "/"!!!
```

Comment:
this I really would like avoid. Users can make mistakes easily and forgetting the last slash in a path is super frequent. I think the code should be changed to let the user specify or not the last slash. Please consider using `pathlib` instead of `os`, it is in the standard python library and it provides support for paths and directories much better than `os` does. Or, at least use `os.path.normpath` which will give you a normalized path with no trailing slash.

Also, it is not written, but probably implicit. Should these paths be absolute or relative?

Reply:
Thanks, this is a very useful hint. I am using pathlib now and the users can type the data paths with or without the slash at the end.